common/src/main/java/gov/cms/ab2d/common/model/Contract.java

```diff
@@ -119,4 +119,25 @@ public boolean updateAttestation(boolean attested, String attestationDate) {
         attestedOn = OffsetDateTime.parse(dateWithTZ, FORMATTER);
         return true;
     }
+
+
+    public boolean isSyntheaContract() {
+        return isSyntheaContract(this.contractNumber);
+    }
+
+    public boolean hasTestDateIssues() {
+        return hasTestDateIssues(this.contractNumber);
+    }
+
+    public static boolean hasTestDateIssues(String contractNumber) {
+        return isTestContract(contractNumber) && !isSyntheaContract(contractNumber);
+    }
+
+    public static boolean isTestContract(String contractNumber) {
+        return contractNumber.startsWith("Z");
+    }
+
+    public static boolean isSyntheaContract(String contractNumber) {
+        return contractNumber.startsWith("Z1");
+    }
```

Comment:
There are at least three approaches to determining the "type" of a contract:
1) Naming Conventions
2) updateMode currently flags TEST contracts.  A new enum type of SYNTHEA could just as easily be added and then this field would server two purposes.
3) add a new enum/field in contract.

For this review, just indicate why #1 was preferred.  The downside is a reduced flexibility in what a contract name could be and a bit of tribal knowledge instead of things being quite explicit.